doc/v2/dev/index_en.rst

```diff
@@ -1,9 +1,27 @@
 Development
 ------------
 
+
+PaddlePaddle adheres to the following three sections of code and document specifications.
+
+
+PaddlePaddle uses git for version management and docker is used for building and testing environment. The code includes Cuda, C++, Python, Shell and other programming languages，which comply with Google C++ Style, Pep-8, and the code base includes style checking by an automatic inspection tool. Code comments need to follow the Doxygen specification, and if code that does not meet the style requirements will fail to compile with. we provide the following guidelines for the use of git, build tests and code development.
```

Comment:
version management -> version **control**

Reply:
docker -> **Docker**

Reply:
Doxygen specification, and if code that does not meet the style requirements will fail to compile with.  -> Doxygen specification. The code that does not meet the style requirements will fail to compile.

Reply:
we provide the -> **We** provide the

Reply:
git -> **Git**